tests/test_bounds.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+# This script is a draft. Note: Bash is not available on CheriBSD.
```

Comment:
Does this script need bash? Ideally we'd stick to the portable core (I, for one, do not use bash!).

Reply:
The comment is to justify some "portable statements" which, in my opinion, are ugly and unreadable compared to Bash 4.x :)